### DISCUSSION TITLE
Is it possible to change the simpleusers landing page from profile to webmail

### DISCUSSION THREAD

**Discussion by mjarvisal** [modoboa/modoboa#3106] [Q&A] [answered]
Every time I or other users log into modoboa web interface the default landing page is the profile page even after changing the initial password.

It would be nicer if after changing the initial password or reviewing and changing the profile settings the default landing page for simpleusers would be the webmail page.

**Comment by Spitfireap** [comment]
It would be useful indeed but it will likely not be put to the current interface since we focus the development on the new one (new-admin). 
We are waiting for vuetify to be updated before starting the development of the user dashboard. Would be a nice feature to have on this one though!

**Comment by tonioo** [comment]
@mjarvisal There is a parameter for that. Look for something called "Top redirection" in the Parameters section

**Reply by mjarvisal** [reply, marked as answer]
@tonioo Found it! It was in the new admin->Settings->General->Miscellaneous->Default top redirection
